pyemma/msm/estimators/bayesian_msm.py

```diff
@@ -132,6 +132,10 @@ def __init__(self, lag=1, nsamples=100, nsteps=None, reversible=True,
             may thus separate the resulting transition matrix. The default
             evaluates to 1/nstates.
 
+	 core_set : None (default) or array like, dtype=int
+		    * If set to None, replaces the state -1 and performs a milestone counting. If no state is
```

Comment:
sentence? "If set to None, replaces state -1 (if applicable) and performs milestone counting. No effect for Voronoi-discretized trajectories (default)."